### DISCUSSION TITLE
27.08.02 Progress Meeting

### DISCUSSION THREAD

**Discussion by ssutjipto** [Woolfrey/software_robot_library#132] [Meetings]
#### Chair:
@ssutjipto 

#### Attendees:
@Woolfrey 
@ssutjipto 
_____

## Announcements
- Shoudong visited Leeds last week, has a PhD student who is keen to work on a control problem

## Agenda
_Note: Last week's actions become this week's agenda_

| Item | Responsible | Issue | Outcome |
|-------|------------------|----------|--------------|
| Integrate the dynamic control class with the new changes in the master branch | @Woolfrey | #133 | Hasn't come back to this, currently working on ROS2 code. Has made this a priority for this week |
| Continue testing (after thesis submission) | @ssutjipto | #128 #117 | Just began testing (has been away throughout the last fortnight) but has encountered edge case (with Trajectory class) whilst testing that needs to be addressed |

## Additional Discussion
| Topic | Raised By | Issue | Outcome |
|---------|---------------|---------|--------------|
| Begin using Mojoco as simulation platform | @Woolfrey | N/A | Videos will be added to this discussion |

## Actions
| Action | Responsible | Tag |
|----------|------------------|-------|
| Integrate the dynamic control class with the new changes in the master branch | @Woolfrey | #133 |
| Create issue for trajectory class | @ssutjipto | |
| Continue testing | @ssutjipto |  #128 #117 |

**Comment by Woolfrey** [comment]
MuJoCo simulation with ROS2 node publishing joint states:

[Screencast from 01-08-24 14:14:50.webm](https://github.com/user-attachments/assets/966ab3af-9b02-449c-b17c-95508f83a798)

[Screencast from 01-08-24 14:15:47.webm](https://github.com/user-attachments/assets/afa57561-ac96-428a-8484-75e7190806c2)
